src/lib.rs

```diff
@@ -18,7 +18,6 @@ pub use graph::{
     RawNodes, VisitOrder, VisitOrderReverse, WouldCycle,
 };
 pub use node::Node;
-pub use sample;
```

Comment:
I'd find it cleaner to re-export the dependency, not just to avoid versioning conflicts, but also to provide a clean public API that does not rely on transitive dependencies.